.github/config/linux.json

```diff
@@ -5,16 +5,17 @@
       "linux-x64x11"
     ],
     "targets": [
```

Comment:
there's no need to modify .github/config/linux.json for adding nplb

cobalt/BUILD.gn already has nplb as one of it's targets and we are already building that for linux
https://source.corp.google.com/h/github/youtube/cobalt/+/main:cobalt/BUILD.gn;l=21

Reply:
oh, interesting. 
Where do we specify the tests to be run?

Reply:
This was discussed in the unit tests tech talk last week—the infra team currently just globs *tests binaries and runs those, so won't pick up nplb. You should reach out to the infra team (probably @oxve) to ask about how best to go about this or if they have something in the works

Reply:
Figured out where to add nplb in the .github files.